src/resources/books.ts

```diff
@@ -0,0 +1,88 @@
+import { keys } from 'ts-transformer-keys'
+import { BookSeries, BookFiles, Images, PersonGroups } from '.'
+import { DatePrecision } from '../date'
+import { DateFilter, IsInFilter } from '../filter'
+import { UnionMap } from '../util'
+
+export type Type = 'books'
+
+export type BookType = 'collected-volumes' | 'monograph'
+export const bookTypes = keys<UnionMap<BookType>>()
+
+export type BookBinding = 'softcover' | 'hardcover'
+
+export type BookOnlineAccessType = 'open' | 'free' | 'subscription'
+export const bookOnlineAccessType = keys<UnionMap<BookOnlineAccessType>>()
+
+export type Attributes = {
+  createdAt: Date
+  updatedAt: Date
+
+  bookType: BookType
+  bookSeriesVolume?: number
+
+  titleTex: string
+  tile?: string
+  subtitleTex: string
+  subtitle?: string
+
+  blurbMarkdownTex?: string
+  blurbMarkdown?: string
+
+  isbn?: string
+  eIsbn?: string
+  doi?: string
+
+  publishedAt: Date
+  publishedAtPrecision: DatePrecision
+  onlineAccessType: BookOnlineAccessType
+
+  mscs?: string[]
+  bicSubjectCategories?: string[]
+  keywords?: string[]
+
+  pages?: number
+  binding?: BookBinding
+  printWidthCm?: number
+  printHeightCm?: number
+
+  publisher: string
+  publisherOrganization?: string
+  publisherAddress?: string
+  publisherUrl: string
+
+  copyright?: string
+  copyrightUrl?: string
+  license?: string
+  licenseCode?: string
+  licenseUrl?: string
+}
+
+export type Relationships = {
+  bookSeries: { type: BookSeries.Type; cardinality: '1' }
```

Comment:
I think a book might not be part of a series? For the frontend we said we make the ui like there was a series "without-series" is that how the data is handeled too?

Reply:
You're right!